invisible_cities/cities/detsim_source.py

```diff
@@ -0,0 +1,22 @@
+import tables as tb
+import pandas as pd
+
+from typing import List
+from typing import Generator
+
+from invisible_cities.io.mcinfo_io import load_mchits_df
+
+#######################################
+############### SOURCE ################
+#######################################
+def load_MC(files_in : List[str]) -> Generator:
+    for filename in files_in:
```

Comment:
here shouldn't we have `try except` in case some of the input files are corrupted/empty?

Reply:
Good point

Reply:
In case the file is empty, you will loop through nothing, so you will get an empty output file.